### ISSUE TITLE
Tuya switch - Expected BEGIN_OBJECT but was STRING at line 1 column 1 path $

### ISSUE THREAD

**Issue by owl770** [smarthomej/addons#588]
I am running V4.1.0 of the org.smarthomej.binding.tuya addon on openHAB V4.1.2. Everything was working fine until there was a power failure. Once the system came back up, one of my switches stopped working. Here is the TRACE log...
```
2024-09-01 13:39:45.766 [TRACE] [.internal.local.handlers.TuyaDecoder] - 48324655dc4f22a73b6d/192.168.68.205:6668: Received encoded '000055AA00000000000000080000006B00000000332E33000000000000000B00000001801735525241CFF59658243689F5761F6E4F9026ED40DE7FEE767B9701146CA70F8C40221AFF138358EBA622A72BD0585904A334A4F034D762CFA2D5012AD4DADC9E944C069FF29AE64A7CAD7812CD1DF0A47FA90000AA55'
2024-09-01 13:39:45.768 [TRACE] [.internal.local.handlers.TuyaDecoder] - 48324655dc4f22a73b6d/192.168.68.205:6668: Decoded raw payload: 592714D4F2080824B013373C25194228846F98BDC647863DF3EFAB581EB296B73DBB2D9139EEA05103DC59A134969F49CA1B30C274C7DCE1193A9C307E0ADD830227A5B1604C9FD99EAD12E32D853A64
2024-09-01 13:39:45.770 [WARN ] [.internal.local.handlers.TuyaDecoder] - 48324655dc4f22a73b6d/192.168.68.205:6668 failed to parse JSON: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1 path $
2024-09-01 13:39:50.177 [TRACE] [rnal.local.handlers.HeartbeatHandler] - 48324655dc4f22a73b6d/192.168.68.205:6668: Sending ping
2024-09-01 13:39:50.179 [DEBUG] [.internal.local.handlers.TuyaEncoder] - 48324655dc4f22a73b6d/192.168.68.205:6668: Sending HEART_BEAT, payload {devId=48324655dc4f22a73b6d, uid=48324655dc4f22a73b6d, t=1725161990, dps=, gwId=48324655dc4f22a73b6d}
2024-09-01 13:39:50.181 [TRACE] [.internal.local.handlers.TuyaEncoder] - 48324655dc4f22a73b6d/192.168.68.205:6668: Sending encoded '000055AA000000CB0000000900000097332E330000000000000000000000003FDE8D467B04D25925B5C6F04BBAA0D1ADB39EB53DEF643EDA6BE571A1FFB38D331F61C799D489BD71DB2232E626490B79A5A7155DAC0220E7FC62FCBC6168F3C9DFA37238C673680028A3BA9EBECAEF8CC3A6F2C97E7FC5D3B7DD002DFDC8B5D4BFAC2F2923294805D51373AA880BD88B8E7740EF798F27C63175A1FEFA12F28E717A580000AA55'
2024-09-01 13:39:50.198 [TRACE] [.internal.local.handlers.TuyaDecoder] - 48324655dc4f22a73b6d/192.168.68.205:6668: Received encoded '000055AA00000000000000090000000C00000000B051AB030000AA55'
2024-09-01 13:39:50.199 [TRACE] [.internal.local.handlers.TuyaDecoder] - 48324655dc4f22a73b6d/192.168.68.205:6668: Decoded raw payload: 
2024-09-01 13:39:50.201 [DEBUG] [.internal.local.handlers.TuyaDecoder] - 48324655dc4f22a73b6d/192.168.68.205:6668: Received MessageWrapper{commandType=HEART_BEAT, content='[B@36d9c3'}
2024-09-01 13:39:50.202 [TRACE] [rnal.local.handlers.HeartbeatHandler] - 48324655dc4f22a73b6d/192.168.68.205:6668: Received pong
```
Any suggestions on how to analyse the ```failed to parse JSON: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1 path $``` problem please?

**Comment by owl770** [comment]
I ended up removing the item and thing from openHAB and adding it back into the Grid Connect app to ensure the switch was working OK. It was. I deleted it from the Grid Connect app and re-discovered it in the Tuya Cloud platform. That worked OK and I was then able to re-discover it in openHAB. It now works OK. The localKey has changed but I think that might happen when you re-pair the device? Not 100% sure. Anyway, the power failure seems to have done 'something bad' to the switch config or openHAB config but removing and re-discovering it has got it back online.
